conduct.md

```diff
@@ -204,7 +220,7 @@ includes members of the Software Engineering class for the spring semester of 20
 
 :heavy_exclamation_mark: TODO: Add an action to be taken for the level I infraction
 
-* Level II: Upon the accumulation of three level I infractions, project managers and the instructor will issue a formal level II infraction.
+Student will be given a written warning from the instructor with a brief description of the infraction they broke and what more appropriate behavior would be and an option to meet with the instructor to discuss alternatives to their action.
 
```

Comment:
I think it would be best to preserve the original text for Level II, while having what your group has proposed here for the action.